test/phoenix_live_view/html_formatter_test.exs

```diff
@@ -393,14 +399,58 @@ defmodule Phoenix.LiveView.HTMLFormatterTest do
     assert_formatter_output(input, expected)
   end
 
+  test "format when there are curly interpolations" do
+    input = """
+      <section>
+        <p>pre{@user.name}pos</p>
+        <p>pre { @user.name}pos</p>
+        <p>pre{@user.name } pos</p>
+        <p>pre { @user.name } pos</p>
+      </section>
+    """
+
+    expected = """
+    <section>
+      <p>pre{@user.name}pos</p>
+      <p>pre {@user.name}pos</p>
+      <p>pre{@user.name} pos</p>
+      <p>pre {@user.name} pos</p>
+    </section>
+    """
+
+    assert_formatter_output(input, expected)
+  end
+
+  test "migrates from eex to curly brackets" do
+    input = """
+      <section>
+        <p><%= @user.name %></p>
+        <p><%= "{" %></p>
+        <script>window.url = "<%= @user.name %>"</script>
+      </section>
+    """
+
+    expected = """
+    <section>
+      <p>{@user.name}</p>
+      <p><%= "{" %></p>
```

Comment:
oh if we keep this intentionally, please disregard this and my previous comment.